product page in build section (#264)
wormhole-foundation/wormhole-docs: build/start-building/products.md

---

> title: Products

Reviewer: This is for SEO purposes, so I think it would benefit from something more descriptive. I think that "products" will get lost in the search results

> description: Compare Wormhole products by features, asset support, integration complexity, and use cases. Explore additional tools for data, governance, and execution.

Reviewer: I feel like we can do better here

::end-spantable::

</div>

> ## Other Products

Author: I could't come up with anything else - maybe more products?

> ## Other Products

Reviewer: Let's split it up like we're doing in the rest of the docs for the new IA: 

```
## Real-time Data

Queries

## Multichain Governance

MultiGov
```

Beyond asset transfers, Wormhole provides additional tools for cross-chain data, governance, and advanced messaging.

> - [**Queries**](/docs/build/applications/queries/overview/){target=\_blank} - a data retrieval service to fetch on-chain data from multiple networks, powering analytics and dashboards. Best for applications that need to simplify multi-chain analytics, reporting, and data aggregation 

Reviewer: ```suggestion
- [**Queries**](/docs/build/applications/queries/overview/){target=\_blank} - a data retrieval service to fetch on-chain data from multiple networks. Best for applications that need multichain analytics, reporting, and data aggregation
```

"powering analytics and dashboards" felt like more of a use case than a description here. So I think it makes more sense to remove that

Reviewer: I tried to come up with something to move "dashboards" to the following line, but I feel like it's implied that with analytics, reporting, and data aggregation that you can create dashboards 🤷‍♀️ - wdyt?

> - [**MultiGov**](/docs/learn/governance/overview/){target=\_blank} - a unified governance framework to manage multi-chain protocol governance through a single mechanism. Best for projects managing multi-chain governance and protocol updates

Reviewer: ```suggestion
- [**MultiGov**](/docs/learn/governance/overview/){target=\_blank} - a unified governance framework that manages multichain protocol governance through a single mechanism. Best for projects managing multichain governance and protocol updates
```

> - [**Wormhole Settlement**](/docs/learn/messaging/wormhole-settlement/overview/){target=\_blank} - a next-generation suite of intent protocols enabling fast multi-chain transfers, optimizing liquidity flows and interoperability without relying on traditional bridging methods. Best for institutional-scale volume and chain abstraction for application developers 

Reviewer: ```suggestion
- [**Wormhole Settlement**](/docs/learn/messaging/wormhole-settlement/overview/){target=\_blank} - a next-generation suite of intent protocols enabling fast multichain transfers, optimizing liquidity flows and interoperability without relying on traditional bridging methods. Best for institutional-scale volume and chain abstraction for application developers 
```

::end-spantable::

</div>

> ## More Products

Reviewer: Let's split it up like we're doing in the rest of the docs for the new IA: 

## Real-time Data

Queries

## Multichain Governance

MultiGov
